_docs-sources/foundations/landing-zone/prerequisites.md

```diff
@@ -2,12 +2,6 @@
 
 The first step to using Gruntwork Landing Zone is to use AWS Control Tower to create a new multi-account setup, which requires that you meet the prerequisites below.
 
-## Gruntwork Pipelines
-
-In order to get the full functionality of Gruntwork Landing Zone, you must use [Gruntwork Pipelines](/pipelines/overview/).
-
-## Enabling AWS Control Tower
-
 To use AWS Control Tower you must first enable it, which requires that you have the following:
```

Comment:
I recommend cutting this sentence entirely, as it's now rather redundant with the intro sentence above.